go/kbfs/kbfsgit/start.go

```diff
@@ -25,6 +25,9 @@ type StartOptions struct {
 	// GitDir is the filepath leading to the .git directory of the
 	// caller's local on-disk repo.
 	GitDir string
+	// LFS indicates whether we should listen for LFS commands instead
+	// of normal git commands.
+	LFS bool
```

Comment:
Why use a bool here instead of a `runnerProcessType`?

Reply:
Because it needs to be set outside this package, and it didn't seem worth exposing that unexported type and all of its enums, when all the world outside this package cares about is boolean.  If you feel strongly though, I can revisit that.